packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -145,87 +131,181 @@ export default {
 <style lang="scss" module>
 
 $font-sizes: (
-    fontSizeSmall: (
+    sm: (
         title: heading-s,
         text: body-s,
     ),
-    fontSizeMedium: (
+    md: (
         title: heading-m,
         text: body-l,
     ),
-    fontSizeLarge: (
+    lg: (
         title: heading-l,
         text: subheading-s,
     ),
-    fontSizeXLarge: (
+    xl: (
         title: heading-xl,
         text: subheading-s,
     ),
-    fontSizeXXLarge: (
+    xxl: (
         title: heading-xxl,
         text: subheading-l,
     )
 );
 
+$flexMargin: spacing(x3);
+
 .c-mediaElement {
     display: flex;
     width: 100%;
+}
+
+.c-mediaElement--col--reverse {
+    flex-direction: column-reverse;
 
-    &.c-mediaElement--reverse {
-        flex-direction: row-reverse;
+    & .c-mediaElement-content {
+        margin-top: $flexMargin;
     }
 }
+.c-mediaElement--col {
+    flex-direction: column;
 
-/**
- * Modifier – .c-mediaElement--stackOnNarrow
- *
- * Applies flex direction column on <=narrow
- */
-@include media('<=narrow') {
-    .c-mediaElement--stackWhenNarrow {
+    & .c-mediaElement-content {
+        margin-bottom: $flexMargin;
+    }
+}
+.c-mediaElement--row {
+    flex-direction: row;
+}
+.c-mediaElement--row--reverse {
+    flex-direction: row-reverse;
+}
+
+
+@each $name, $value in $breakpoints {
+
+    /**
+    * Modifier – .c-mediaElement--gte--$name--col
+    * Modifier – .c-mediaElement--gte--$name--col--reverse
+    * Modifier – .c-mediaElement--gte--$name--row
+    * Modifier – .c-mediaElement--gte--$name--row--reverse
+     *
+     * Applies flex direction column on $name
+     */
+    @include media('>='+$name) {
```

Comment:
I feel like you could nest these another level deeper to avoid the repetition, but it might be already a bit too clever to abstract any further!

E.g. 
```scss
$operators: (
    'lte': '<=',
    /* ... */
)

@each $name, $value in $breakpoints {
    @each $operator, $operatorClassAbbreviation in $operators {
        @include media($operator+$name) {
            .c-mediaElement-col--#{$operatorClassAbbreviation}--#{$name} { /* ... */ }
            /* ... */
        }
    }
}
```

Reply:
Yea this was the same conversation I had with @ashleynolan tbh. It's a hard one because my developer head tells me it can be abstracted further, but from a readability point it may make it harder to understand. What's your thoughts?

Reply:
Yeh, I'm on the side of having this listed out (just as it doesn't impact performance in any way, as it compiles out to the larger code block), but interested in hearing people's thoughts.

Reply:
For now I will leave as is and we can come back and adjust if we are finding the repetition annoying when doing any future updates.